tools/code-analyzer/README.md

```diff
@@ -8,22 +8,21 @@
 
 Currently there are just 2 commands:
 
-1. `analyzer`. Analyzer serves 2 roles currently, as a linter for PRs to check if introduced hook/template/db changes have associated changelog entries and also to provide file output of changes between
-WooCommerce versions for the purpose of automating release processes (such as generating release posts.)
+1. `lint`. Analyzer is used as a linter for PRs to check if hook/template/db changes were introduced. It produces output either directly on CI or via GH actions `set-output`.
```

Comment:
So this is not meant to be used as a CLI output anymore? I don't think it was too useful as purely a CLI tool since ultimately its consumed by a GH action and a release post generator, but worth bringing up because its not readable anymore. But maybe thats ok, what do you think?

<img width="968" alt="image" src="https://user-images.githubusercontent.com/1922453/189042031-20f1f350-3a47-40fa-84c9-ddb72fc5c293.png">

Reply:
@psealock it defaults to github output but I've retained the CLI output if anyone needs it, you just set the `-o cli` flag.

As you say its not super useful any more which is why I've made `github` the default.

Reply:
This prompted me to find a bug in the output setting so thanks! I've made cli the default and passed `-o` to the gh action here https://github.com/woocommerce/woocommerce/pull/34600/commits/caa6d5f6e8a684d8510e07b8503d98366c2a1afb